### PR TITLE
New payment process and various website fixes

### PR DESCRIPTION
**Pull request by ecino**

### CODE REVIEW

File: muskathlon/templates/muskathlon_my_home.xml [hunk 1]
```diff
@@ -50,7 +28,8 @@
                     </t>
                     <div id="modal_flight_details" class="modal fade cms_modal_form" tabindex="-1" role="dialog"
                          aria-labelledby="donation_modal_label" aria-hidden="true">
-                        <div class="modal-dialog modal-xl" role="document">
+                        <!-- TODO Replace by modal-xl when Bootstrap 4.2 is supported by Odoo -->
+                        <div class="modal-dialog modal-lg" role="document">
```
Comment (quentingigon) [inline muskathlon/templates/muskathlon_my_home.xml:32]: Do we really rollback this change ? Don't we use bootstrap 4.2 now ?

Comment (ecino) [reply muskathlon/templates/muskathlon_my_home.xml:32]: Yes I checked and Odoo 12 uses bootstrap 4.1.3 unfortunately :cry:

**Review by quentingigon — COMMENTED** [review]
only one remark

File: muskathlon/templates/muskathlon_my_home.xml (hunk 1 above)
Comment (quentingigon) [reply muskathlon/templates/muskathlon_my_home.xml:32]: Ok, too bad

**Pull request merged**